hudi-utilities/src/main/java/org/apache/hudi/utilities/streamer/HoodieStreamer.java

```diff
@@ -150,21 +159,27 @@ public HoodieStreamer(Config cfg, JavaSparkContext jssc, FileSystem fs, Configur
   public HoodieStreamer(Config cfg, JavaSparkContext jssc, FileSystem fs, Configuration conf,
```

Comment:
Use TestHoodieDeltastreamer class if required.

Reply:
not needed now that we removed the extra code